db/migrations/8.4.1.sql

```diff
@@ -0,0 +1,65 @@
+with asset_data as (
```

Comment:
@prashantasdeveloper I think you took 8.4.2 in master branch.

I want to merge this into alpha so we can test resync. Does this need to get 8.4.3 now, or will filling in the "hole" be OK?

Reply:
This would require 8.4.3